src/main/java/com/kodilla/ecommercee/entity/ProductMock.java

```diff
@@ -0,0 +1,23 @@
+package com.kodilla.ecommercee.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+public class ProductMock {
```

Comment:
Nazwij to po prostu Product, Ktoś kto będzie miał zadanie na Product po prostu uzupelni resztę pól